### PR TITLE
Issue #14: Expected tech (section 10)

### PR DESCRIPTION
**Pull request by ianzachary**
﻿## GitHub Issue

Issue #14

## Description

Added section 10 expected technology to the DevelopmentPlan.tex.

## How I tested

Locally compiled changes successfully and viewed in PDF.

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -188,37 +188,51 @@ \section{Proof of Concept Demonstration Plan}
...
+  \hline
+  Project Management & GitHub Projects & Used for Kanban board, task assignment, backlogging, etc. \\
+  \hline
+  Back-End Development & C/C++, MATLAB & Possible specific languages, high performance and speed is required for
+  signal processesing.\\
+  \hline
+  Sound File Conversion & libsndfile  & A specific library that was researched and will likely be used
+  for sampled sound file format conversion.\\
+  \hline
+  Code Formatting & Linter(s) &  Specific tools are ndetermined at this stage as it is language dependant.
```
Comment (emilyperica) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:208]: ndetermined -> undetermined

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -188,37 +188,51 @@ \section{Proof of Concept Demonstration Plan}
...
+  Back-End Development & C/C++, MATLAB & Possible specific languages, high performance and speed is required for
+  signal processesing.\\
+  \hline
+  Sound File Conversion & libsndfile  & A specific library that was researched and will likely be used
+  for sampled sound file format conversion.\\
+  \hline
+  Code Formatting & Linter(s) &  Specific tools are ndetermined at this stage as it is language dependant. 
+  To be used with CI workflows to format code.\\
+  \hline
+  Testing & MATLAB unittest/automation, CMake, Unity, etc.  & Undetermined, likely a unit testing framework
```
Comment (emilyperica) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:211]: Why unity? And when you say undetermined, does this mean the description itself is undetermined?

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 3]
```diff
@@ -188,37 +188,51 @@ \section{Proof of Concept Demonstration Plan}
...
+  \hline
+  Musical Instruments & Keyboard, guitar, ukelele, etc. & Used to generate input signals.\\
+  \hline
+\end{tabular}
+\endgroup
+\subsection{Continuous Integration Plans}
+GitHub Actions (GHA) will be used for continuous integration. They will employ automated workflows for various unit and integration tests
+as well as code formatting/linting. One specific workflow that will be created is .tex to .pdf format conversion to ease documentation editing.
+GHA will also ideally be used to test code coverage metrics. One of the major components of the project is the signal processing algorithm, 
+depending on the implementation details, different metrics in addition to the basic ones such as statement coverage. For example, the designed algorithm
```
Comment (emilyperica) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:234]: The sentence starting with 'One of the major...' is hard to understand, maybe split it into two sentences?

**Review by emilyperica — CHANGES_REQUESTED** [review]

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 2 above)
Comment (ianzachary) [reply docs/DevelopmentPlan/DevelopmentPlan.tex]: Sorry, should have been more specific. I meant Unity Test, a C testing tool. It's mainly used for embedded systems but I wanted to include it as an example to be a bit broad in the possible choices.

**Review by emilyperica — DISMISSED** [review]
LGTM

**Review by emilyperica — APPROVED** [review]

**Pull request merged**